### DISCUSSION TITLE
Cannot display plantuml diagrams on markdown pages, but it works during edition

### DISCUSSION THREAD

**Discussion by golflima** [requarks/wiki#3623] [Error / Bug Report]
**Describe the bug**

We enabled the PlantUML renderer, and we use the default settings for it (using server `https://www.plantuml.com/plantuml`).
When we create a Mardown page using a PlantUML diagram (with the block `@startuml`), we see on the editor the preview of the diagram. But when the page is saved, we only see the source code of the diagram.

**To Reproduce**

1. Create a new page using Markdown, when PlantUML renderer is enabled
2. Save it
3. Edit it, add a PlantUML diagram bloc, the preview renders it correctly while editing it
4. Save the page
5. View it: the PlantUML is rendered as a normal source code block, not as a diagram anymore.

**Expected behavior**

See the PlantUML rendered when viewing the page.

**Logs**

On the view page, I saw that the script `https://wiki.domain.tld/_assets/js/prism/prism-plantuml.min.js` was not found, it throws a 404. I checked in the container, and this file doesn't exists in the `/wiki/assets/js/prism/` folder. Maybe an asset is missing in the Docker image?

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: `requarks/wiki:2.5.170`
 - Database engine: `postgres:12.2`

Thank you in advance guys for your support :)

**Comment by fr34kyn01535** [comment]
Issue seems to persist, same here with Wiki.js 2.5.170 on Docker.

**Reply by ckazimie** [reply]
Same problem Wiki.js 2.5.296
